### DISCUSSION TITLE
Separate dependencies that are used in the pipeline: pipelineDependencies

### DISCUSSION THREAD

**Discussion by abelmokadem** [npm/feedback#295] [CLI] [answered]
I like the concept of devDependencies and dependencies, nothing to say about that. But I see a need for a third type of dependencies. Dependencies that are solely used and installed in your CI/CD pipelines. 

```
...
pipelineDependencies: {
    "semantic-release": "^17.4.0",
}
...
```

It would be nice to have these dependencies separated from devDependencies and dependencies.

My biggest argument for this is: speed on developers machine (faster install, less files, faster indexing for editors)

**Comment by MylesBorins** [comment, marked as answer]
Hey @abelmokadem. I know that the CLI team has historically not been looking at adding additional dependency types. With that said, if you would like to pursue proposing this new feature the appropriate path would be to open an issue over at https://github.com/npm/rfcs and follow the template provided.
